pkg/storaged/overview.jsx

```diff
@@ -47,11 +68,14 @@ export class OverviewSidePanel extends React.Component {
                            not_installed_text={this.props.not_installed_text}
                            install_title={this.props.install_title}>
                 { this.props.children.length > 0
-                    ? <table className={"table" + (this.props.hover !== false ? " table-hover" : "")}>
-                        <tbody>
-                            { this.props.children }
-                        </tbody>
-                    </table>
+                    ? <React.Fragment>
+                        <table className={"table" + (this.props.hover !== false ? " table-hover" : "")}>
+                            <tbody>
+                                { children }
+                                { show_all_button }
+                            </tbody>
+                        </table>
+                    </React.Fragment>
```

Comment:
The Fragment is't necessary here (probably a leftover from an intermediate version?), but also harmless.

Reply:
Correct.